### PR TITLE
add note on structure-icons mapping

### PR DESCRIPTION
**Pull request by Dhaulagiri**
## :pushpin: Summary

Adds a brief note that there is a file containing mappings between structure icon names and flight icon names

### CODE REVIEW

**Review by amyrlam — APPROVED** [review]
:shipit:

**Comment by didoo** [comment]
Should we add the same not in the website too? At the end, that's where most of the people will land anyway to look for icons (while the README is buried in the repo).

File: ember-flight-icons/tests/dummy/app/templates/engineering.hbs [hunk 1]
```diff
@@ -262,4 +262,21 @@
...
+    Updating Existing Interfaces
+  </h2>
+
+  <ul>
+    <li>
+      We maintain
+      <a href="https://github.com/hashicorp/flight/blob/main/structure-mappings.json" rel="external" class="ds-a">a
+        mapping of icon names between Structure and Flight</a>
+      that can be referenced to migrate an icon from Structure to Flight. It is also possible to write codemods to
+      automate this migration. If you are interested in learning more about this reach out in #team-design-systems.
```
Comment (didoo) [inline ember-flight-icons/tests/dummy/app/templates/engineering.hbs:279]:
```suggestion
      automate this migration. If you are interested in learning more about this reach out in [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS).
```

**Review by didoo — APPROVED** [review]

File: ember-flight-icons/tests/dummy/app/templates/engineering.hbs (hunk 1 above)
Comment (didoo) [reply ember-flight-icons/tests/dummy/app/templates/engineering.hbs:279]: @Dhaulagiri a small change, but otherwise LGTM!

**Pull request merged**